app/helpers/ems_physical_infra_helper/textual_summary.rb

```diff
@@ -14,7 +14,7 @@ def textual_group_properties
   def textual_group_relationships
     TextualGroup.new(
       _("Relationships"),
-      %i(physical_servers datastores vms hosts)
+      %i(physical_servers datastores vms physical_servers_with_host)
```

Comment:
This looks like you can no longer see `hosts`? Was that a copypaste that was never alive or is it a bug here?